packages/relay-signer/README.md

```diff
@@ -1,5 +1,303 @@
 # Defender SDK Relay Signer Client
 
-The OpenZeppelin Defender provides a security operations (SecOps) platform for Ethereum with built-in best practices. Development teams implement Defender to ship faster and minimize security risks.
+Defender Relay Signer Client lets you send transactions to any supported network using private relayers. Each relayer has its own secure private key, and a set of API keys. You can send transactions via your relayers by POSTing to the Defender HTTP API, or using this library.
```

Comment:
@MCarlomagno could you please review docs in this readme file to verify that nothing is off. Thanks